lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -119,6 +131,13 @@ def get_course_hierarchy(self):
                     {'#type': 'vertical', '#ref': 'G'},
                 ],
             },
+            {
+                '#type': 'sequential',
+                '#ref': 'NotASpecialExam',
```

Comment:
And, while you're there, can you update the "Exam1" and "Exam2" names to "TimedExam" and "PracticeExam" respectively?